### ISSUE TITLE
Decorative element in footer being read as "four pointed star"

### ISSUE THREAD

**Issue by rebekahmarkowitz** [WordPress/twentytwentyfour#45]
## Issue summary

Decorative element on the footer is reading as "black four pointed star" on screen readers

## Steps to reproduce
Implement voice over on macbook pro in chrome.

The decorative element in the below screen shot of the homepage in the footer is reading as "black four pointed star"

<img width="1362" alt="Screenshot 2023-08-24 at 3 19 34 PM" src="https://github.com/WordPress/twentytwentyfour/assets/93548879/b938fa72-39ce-4a2f-81c8-07ef6f0d4998">

## Recommendation
Star in footer should be an image with no alt text vs just text.

## Specifications
Template File: Homepage Footer
**WCAG SC:** [1.1.1](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)

**Comment by sofiashendi** [comment]
I created this PR to fix the issue: #51

**Comment by jessynd** [comment]
Great fix, @sofiakanopi  - I just posted an [issue](#56) with the corresponding diamond icon in the header and there I think an image element with an empty alt property is more appropriate instead of an `aria-hidden="true"` because of how users might interpret that as the place where a logo might go. Since the two are connected, I wonder if the same solution should apply in both the header and the footer? What do you think?

**Comment by sofiashendi** [comment]
@jessynd that's a great question! I normally tend to stay away from images if text is an option, as it's lighter and does not need to request another file on page load.

**Comment by MaggieCabrera** [comment]
Closing this since we are using the site logo now
